cmd/snap-bootstrap/cmd_scan_disk.go

```diff
@@ -106,7 +106,37 @@ func isGpt(probe blkid.AbstractBlkidProbe) bool {
 	return pttype == "gpt"
 }
 
-func probePartitions(node string) ([]Partition, error) {
+func probeFilesystem(node string) (Partition, error) {
+	var p Partition
+
+	probe, err := blkid.NewProbeFromFilename(node)
+	if err != nil {
+		return p, err
+	}
+	defer probe.Close()
+
+	probe.EnableSuperblocks(true)
+	probe.SetSuperblockFlags(blkid.BLKID_SUBLKS_LABEL | blkid.BLKID_SUBLKS_UUID)
+
+	if err := probe.DoSafeprobe(); err != nil {
+		return p, err
+	}
+
+	val, err := probe.LookupValue("LABEL")
+	if err != nil {
+		return p, err
+	}
+	p.Name = val
+
+	// UUID is not required as a part of fallback mode
+	val, err = probe.LookupValue("UUID")
+	if err == nil {
+		p.UUID = val
```

Comment:
Please, do not put the filesystem uuid in there. This is for the partition UUID. Do not mix them up or it will be very difficult to understand what is happening.